src/init.c

```diff
@@ -88,7 +88,7 @@ void __attribute__ ((constructor)) icainit(void)
 		return;
 
 	if (stats_mmap(-1) == -1) {
-		syslog(LOG_INFO,
+		syslog(LOG_WARNING,
```

Comment:
This might happen when running under a sandbox. So now we will see warnings instead of informational messages. Probably OK, it might just cause people to see/recognize this message more often now than before. I guess we just need to be aware about this.